<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDRegister</class>
 <widget class="QMainWindow" name="WDRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tMail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>手机号/邮箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tPasswordRepeat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用户注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tNickName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNotice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
